library/src/morpheus-master/sass/base/_defaults.scss

```diff
@@ -137,13 +139,13 @@ input[type="submit"],input[type="button"],input[type="reset"] {
 }
 
 input[type="checkbox"], input[type="radio"]{
-	background-color: #FFF;
-	border: 1px solid #A0A0A0;
+	background-color: var(--sakai-background-color-1);
+	border: 1px solid var(--sakai-text-color-1);
+	color: var(--sakai-text-color-1);
 	display: inline-block;
 	height: 16px;
 	width: 16px;
 	vertical-align: sub;
-	@include appearance(none);
```

Comment:
This change and line 172 below result in sakai now using the browser native checkbox/radio button. I prefer this solution to having multiple images for each theme's checkmark. In the future, we can write custom form elements if we wish but I thought it out of scope for this PR.

Reply:
There seems to be a pretty common pattern for custom checkboxes/radio buttons but I thought it outside the scope of this PR. I saw several articles linking to this method https://www.w3schools.com/howto/howto_css_custom_checkbox.asp